includes/class-wporg-notifications.php

```diff
@@ -402,17 +359,9 @@ private static function get_project_to_translate( WP_Comment $comment ): GP_Proj
 	private static function get_main_projects():array {
 		global $wpdb;
 
-		$main_projects = $wpdb->get_results(
-			$wpdb->prepare(
-				"
-			SELECT id
-			FROM {$wpdb->gp_projects}
-			WHERE parent_project_id IS NULL"
-			),
-			ARRAY_N
-		);
+		$main_projects = $wpdb->get_col( "SELECT id FROM {$wpdb->gp_projects} WHERE parent_project_id IS NULL" );
 
-		return array_merge( ...$main_projects );
```

Comment:
Didn't quite understand why this is necessary, also array spread is only PHP 7.4.

Reply:
I am not sure about this :/ 
Array spread will work because we use PHP 7.4 at DotOrg, but the `array_merge` is not needed.